microk8s-resources/actions/common/utils.sh

```diff
@@ -948,3 +948,24 @@ wait_for_default_route() {
     sleep 2
   done
 }
+
+is_ec2_instance() {
```

Comment:
We could place this function on the master branch so we get it for free on the next eksd release.